api/api.go

```diff
@@ -1,7 +1,26 @@
 package api
 
-import "go.sia.tech/core/types"
+import (
+	"go.sia.tech/core/types"
+	"go.sia.tech/siad/v2/wallet"
+)
 
 type WalletBalance struct {
 	Siacoins types.Currency `json:"siacoins"`
 }
+
+type Address struct {
+	Address types.Address `json:"address"`
+}
+
+type Addresses struct {
+	Addresses []types.Address `json:"addresses"`
+}
+
+type Transactions struct {
+	Transactions []wallet.Transaction `json:"transactions"`
+}
+
+type SendReceipt struct {
+	Error *error `json:"success"`
+}
```

Comment:
For API errors, we use `http.Error` -- see examples in e.g. https://github.com/lukechampine/walrus/blob/master/server.go